paracelsus/transformers/mermaid.py

```diff
@@ -29,15 +30,16 @@ def _column(self, column: Column) -> str:
                 column_str += " PK"
         elif len(column.foreign_keys) > 0:
             column_str += " FK"
+            options.append(f"Foreign key references {column.table.name}")
```

Comment:
Can you make this a separate pull request? I'm not sure I want this comment in there, especially when we have the relationships drawn out already with the diagram itself.